package.json

```diff
@@ -24,7 +24,7 @@
     "body-parser": "^1.18.2",
     "ch-logging": "git+ssh://git@github.com/companieshouse/ch-structured-logging-node.git#a3e0dcc57f91f63803e02ceef2cc5e4038de9a19",
     "ch-node-session-handler": "git+ssh://git@github.com/companieshouse/node-session-handler.git#3.0.1",
-    "ch-sdk-node": "git+ssh://git@github.com/companieshouse/ch-sdk-node.git#0.2.10",
+    "ch-sdk-node": "git+ssh://git@github.com/companieshouse/ch-sdk-node.git#4f57d67ee531cbc33ede007bc590ef4724c04c20",
```

Comment:
TODO replace hash with version bump once https://github.com/companieshouse/ch-sdk-node/pull/25 is merged

Reply:
Turns out the commit hash is recommended, so resolving this convo